ui/app/components/clients/no-data.hbs

```diff
@@ -7,11 +7,13 @@
   <EmptyState
     @title="No data received {{if @dateRangeMessage @dateRangeMessage}}"
     @message="Tracking is turned on and Vault is gathering data. It should appear here within 30 minutes."
+    class="is-shadowless"
```

Comment:
Moving the tabs below the filters removed the need for the empty state to have a top box shadow, removed it from relevant empty states example:
## before
<img width="1070" alt="Screenshot 2024-04-10 at 5 29 18 PM" src="https://github.com/hashicorp/vault/assets/68122737/0cece147-b06b-4784-8c3c-519fa23e2a6a">

## after
<img width="1084" alt="Screenshot 2024-04-10 at 5 28 40 PM" src="https://github.com/hashicorp/vault/assets/68122737/13a751bb-19b5-486e-9eaa-2edfbecc41f2">